pkg/addon/iampolicy/manifests/managedclusterchart/templates/policy.open-cluster-management.io_iampolicy_crd.yaml

```diff
@@ -82,6 +82,8 @@ spec:
               enum:
               - Inform
               - inform
+              - Enforce
```

Comment:
Could you please also update the description of the property?

Reply:
You mean the `description: enforce, inform`?

I wouldn't change it here, but we could update it in the iam-policy repo: https://github.com/stolostron/iam-policy-controller/blob/main/api/v1/iampolicy_types.go#L56

Reply:
You're right, I had accidentally changed the description in the wrong spot:
https://github.com/stolostron/iam-policy-controller/pull/122